src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -234,21 +237,22 @@ def generate_reports(self, report: Report) -> None:
         for reporter in self.reporters:
             reporter.generate(output_target_path, report)
 
-    def resolve_dependencies(self, main_ctx: AnalyzeContext) -> dict[str, DependencyInfo]:
+    def resolve_dependencies(self, main_ctx: AnalyzeContext, sbom_path: str) -> dict[str, DependencyInfo]:
         """Resolve the dependencies of the main target repo.
 
         Parameters
         ----------
         main_ctx : AnalyzeContext
             The context of object of the target repository.
 
```

Comment:
No need for newline here.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/105/commits/e9d7a5f7cfc3cccfa399875c89958bebe940d4ce